docs/source/1.0/spec/core/lexical-structure.rst

```diff
@@ -333,12 +360,44 @@ Syntactic shape IDs in the IDL
 
 Unquoted string values that are not object keys in the Smithy IDL are
 considered lexical shape IDs and are resolved to absolute shape IDs using the
-process defined in :ref:`relative-shape-id`. Values that are not meant to be
-shape IDs MUST be quoted.
+process defined in :ref:`relative-shape-id`.
+
+The following model defines a list that references a string shape defined
+in another namespace.
+
+.. code-block:: smithy
+
+    namespace smithy.example
+
+    use smithy.other#MyString
 
-For example, the following model resolves the value of the :ref:`error-trait`
-to the shape ID ``"smithy.example#client"`` rather than using the string
-literal value of ``"client"``, causing the model to be invalid:
+    list MyList {
+        member: MyString
+    }
+
+The above model is equivalent to the following JSON AST model:
+
+.. code-block:: json
+
+    {
+        "smithy": "1.0",
+        "shapes": {
+            "smithy.example#MyList": {
+                "type": "list",
+                "members": {
+                    "target": "smithy.other#MyString"
+                }
+            }
+        }
+    }
+
+.. rubric:: Use quotes for literal strings
+
+Values that are not meant to be shape IDs MUST be quoted. The following
```

Comment:
Minor: "Values in Smithy IDL..." might help reset context after just talking about the AST. It looks like this is done below as well.